src/vs/workbench/common/editor/untitledEditorInput.ts

```diff
@@ -199,16 +199,25 @@ export class UntitledEditorInput extends EditorInput implements IEncodingSupport
 	}
 
 	public suggestFileName(): string {
+		var FileName: string;
```

Comment:
Use `let` instead of `var` as a good practice. Also according to the [coding guidelines](https://github.com/Microsoft/vscode/wiki/Coding-Guidelines) `Use camelCase for property names and local variables`